pyaedt/hfss.py

```diff
@@ -373,6 +373,14 @@ def create_frequency_sweep(self, setupname, unit="GHz", freqstart=1e-3, freqstop
         -------
         SweepHFSS
             Sweep object.
+
+        Examples
+        --------
+
+        Create a frequency sweep named 'Sweep' for the analysis setup named 'Setup1'.
+
+        >>> frequency_sweep = aedtapp.create_frequency_sweep("Setup1", sweepname ="FrequencySweep")
```

Comment:
```suggestion
        >>> frequency_sweep = aedtapp.create_frequency_sweep("Setup1", sweepname="FrequencySweep")
```

Reply:
This needs the application setup beforehand as in #286, and we need the `__repr__` for `frequency_sweep`.

Reply:
@akaszynski I plan to get a working example of how doctest setup will work with Icepak and then show @MaxJPRey how it works. He can then update this example for HFSS using the same approach.

Reply:
@akaszynski I am not sure what you are expecting regarding the __repr__ for frequency_sweep.
Do you expect me to directly write something such as:
`>>> aedtapp.create_frequency_sweep("Setup1", sweepname="FrequencySweep")`
?
Or do you want me to print frequency_sweep after declaring/assigning it?

Reply:
> Or do you want me to print frequency_sweep after declaring/assigning it?

Yes, that's ideal.  It's always helpful to see the return of a method, even if it's something trivial.  The `__repr__` for `frequency_sweep` isn't likely to be great, so you could even run `type(frequency_sweep)`, but regardless, showing something is best.

Reply:
Done.